core/src/avm2/regexp.rs

```diff
@@ -148,6 +151,8 @@ impl<'gc> RegExp<'gc> {
         }
     }
 
+    // Helper for replace_string. Evaluates the special $-sequences
```

Comment:
The convention for function documentation comments is triple slashes: `///`.

Reply:
Done.